### ISSUE TITLE
Add Colab notebook for English to Azerbaijani translation using custom NLLB finetune checkpoint

### ISSUE THREAD

**Issue by tayden1990** [tayden1990/GlobGram-alpha-v2.0.0#2]
Create a Google Colab notebook that:
- Mounts Google Drive
- Loads the NLLB fine-tuned checkpoint for English to Azerbaijani translation (as shown in the attached image)
- Prompts the user to upload an English .txt file
- Runs translation on the uploaded file using the model
- Downloads the result as a .txt file with Azerbaijani translation
- Optimizes for A100 80GB GPU

The notebook should:
- Use transformers and PyTorch
- Support model loading from `/content/drive/My Drive/nllb_en_to_azb_finetune/checkpoint-62000`
- Handle large files efficiently
- Set forced BOS token for Azerbaijani (azb_Arab) language in NLLB

Reference:
<img alt="Google Drive checkpoint directory" width="1260" src="https://github.com/user-attachments/assets/ff024330-1f79-4476-a20f-0b075f68b3a9" />

Label as documentation or enhancement.
